source/OutfitterPanel.cpp

```diff
@@ -231,9 +258,10 @@ void OutfitterPanel::DrawItem(const string &name, const Point &point)
 
 
 
-int OutfitterPanel::DividerOffset() const
+double OutfitterPanel::ButtonPanelHeight() const
 {
-	return 80;
+	// The 60 is for padding and the Info lines
```

Comment:
```suggestion
	// The 60 is for padding the credit and cargo space information lines.
```
I guess that's what you meant?